bitnami/redis/templates/scripts-configmap.yaml

```diff
@@ -63,9 +123,26 @@ data:
         export REDIS_REPLICATION_MODE="master"
     else
         export REDIS_REPLICATION_MODE="slave"
+
+        retry_count=1
+        while validate_quorum
+        do
+            info "sleeping, waiting for Redis master to come up"
+            sleep 1s
+            if ! ((retry_count % 11)); then
```

Comment:
Should this recovery feature be behind a config value that is disabled by default? This is pretty much a hack to recover broken clusters automatically, but it could, in theory, cause split brain issues if fired while there's network issues between nodes/pods. I've implemented this because we're going to use it ourselves, and I believe this should fix https://github.com/bitnami/charts/issues/7283 but in most cases, failover should be left to Sentinel's internal logic, and the cluster manually recovered by an operator that triggers the failover in the cluster using the sentinel command.

Reply:
I think that it is a good idea and the users can enable/disable when they want.

Reply:
I've added a new value to enable that feature.